### ISSUE TITLE
Where can I find plug-ins for nomacs?

### ISSUE THREAD

**Issue by Nokia808** [nomacs/nomacs#446]
Hi. Currently, I'm using nomacs from my official Fedora repositories, version 30.
In nomacs, there should be plugins, but wherever I open a plug-in field I see it empty! I searched Fedora repositories but I did not find any thing!

Where can I find plug-ins for nomacs? What are these plug-ins?

Are such plug-ins will be be available for flatpak package on FlatHub that you are already working to add it?

**Comment by diemmarkus** [comment]
the most important plugin (from my point of few) is [painting](https://github.com/nomacs/nomacs-plugins/pull/27) on an image. old linux packages (i.e 3.8) do not ship with plug-ins. But it should be straight forward for the flatpak package (if I ever get this done : )

**Comment by Nokia808** [comment]
@diemmarkus 
On my system version 3.12 but no any plug-in !
However, I saw you achieve success in building of flatpak. Flatpak is better option than system package provided it is packaged by the developer itself not by other entity, which exact case with you, so just when nomacs become available on FlatHub I will install it as flatpak & this way I can be with you in testing new features you adding ....

**Comment by Nokia808** [comment]
@diemmarkus 
Hi. 1st of all, congratulation for success on FlatHub. I was surprised when saw you nearly hopless from get your PR on FlatHub to be merged ..... !

Now, please, before uninstalling nomacs from my official Fedora repositories & install flatpak package, can you issue me that your flatpak package already containing plug-ins or you have to add them at later stage as extension points to your flatpak ?

**Comment by diemmarkus** [comment]
Nope, plugins are not there yet. I can notify you as soon as they are available...

**Comment by scrubbbbs** [comment]
Plugins are a separate repository and package on some distros, and so you have to install them separately. That should end going forward as we merged the plugins repo with nomacs. There is another open issue for flatpak plugins so I'll close this one.

Duplicate: #1047
